src/hash/rpo/digest.rs

```diff
@@ -96,6 +96,36 @@ impl Deref for RpoDigest256 {
     }
 }
 
+impl Ord for RpoDigest256 {
+    fn cmp(&self, other: &Self) -> Ordering {
+        // compare the inner u64 of both elements.
+        //
+        // it will iterate the elements and will return the first computation different than
+        // `Equal`. Otherwise, the ordering is equal.
+        //
+        // the endianness is irrelevant here because since, being this a cryptographically secure
+        // hash computation, the digest shouldn't have any ordered property of its input.
+        //
+        // finally, we use `Felt::inner` instead of `Felt::as_int` so we avoid performing a
+        // montgomery reduction for every limb. that is safe because every inner element of the
+        // digest is guaranteed to be in its canonical form (that is, `x in [0,p[`).
```

Comment:
minor nit: seems like there is a typo in the range. Should probably be `x in [0, p)`.

Reply:
Fixed!